src/mixins/nr-affiliation-mixin.ts

```diff
@@ -198,6 +199,59 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
     }
   }
 
+  /**
+ * Handle "Amalgamate Now" button.
+ * Submit an amalgamation draft depending on business type.
+ * Redirect to Dashboard.
+ * @param legalType The legal type of the IA that's being incorporated.
+ */
+  async amalgamateNow (legalType: CorpTypeCd): Promise<any> {
+    try {
+      // show spinner since this is a network call
+      this.$root.$emit('showSpinner', true)
+      const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
+      const businessId = await this.createBusinessAA(accountId, legalType)
+      this.goToEntityDashboard(businessId)
+      return
+    } catch (error) {
+      this.$root.$emit('showSpinner', false)
+      this.setIncorporateNowErrorStatus(true)
+      throw new Error('Unable to Amalgamate Now ' + error)
+    }
+  }
+
+  /**
+   * Create a draft amalgamate application based on selected business type (If applicable).
+   * @param accountId Account ID of logged in user.
+   * @param legalType The legal type of the IA that's being incorporated.
```

Comment:
- it's a draft "amalgamation application"
- I think there's always a business type, so remove "if applicable"
- on line 226, same fix as line 206